docker-entrypoint.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
-# Add crontab jobs
-echo "Calling script to set up server cronjobs"
-/bin/bash scripts/cron_setup.sh
+if [[ "$USE_CRON" == "true" ]]; then
+    # Add crontab jobs
+    echo "Calling script to set up server cronjobs"
+    /bin/bash scripts/cron_setup.sh
```

Comment:
Recapping our zoom continued! Plan A: hoping we can get the first call to `rake check:all` to run either in the cron job or here. That way, there's a very clear distinction that the page checks happen in cron locally and in the rake task in the deployed environment.

Reply:
Play A was a success with `bundle exec rake check:all` added to the docker_entrypoint